### DISCUSSION TITLE
M2A Configuration Details

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#16367] [Archive]
Currently, my M2A display shows each item as a "bar". 
Instead, I'd like to display the field value and make it directly editable from this page.. Is that possible?

ALSO: The bars currently display a title for `directus_files` and the `id` for the other two. After a few attempts, I couldn't find how/where to reconfigure that display template.

<img width="1468" alt="Screenshot 2022-11-08 at 9 43 31 PM" src="https://user-images.githubusercontent.com/62306630/200733195-80d564b0-e51f-4bbd-8725-012257443508.png">

**Comment by br41nslug** [comment]
> I'd like to display the field value and make it directly editable from this page.. Is that possible?

With the built-in interface this is not possible.

> After a few attempts, I couldn't find how/where to reconfigure that display template.

The display templates used for m2a are the display templates configured per collection in the data-model.

**Reply by erondpowell** [reply]
> The display templates used for m2a are the display templates configured per collection in the data-model.

Pretty sure I tried this, I will try again. I must have done something wrong.
